src/actinia_core/resources/common/response_models.py

```diff
@@ -1150,3 +1150,124 @@ def create_response_from_model(response_model_class=ProcessingResponseModel,
         return pickle.dumps([http_code, resp_dict])
     else:
         return jsonify(resp_dict)
+
+
+class UserListResponseModel(Schema):
+    """Response schema that is used to list all users.
+
+    """
+    type = 'object'
+    properties = {
+        'status': {
+            'type': 'string',
+            'description': 'The status of the request'
+        },
+        'user_list': {
+            'type': 'array',
+            'items': {'type': 'string'},
+            'description': 'The names of all users'
+        }
+    }
+    required = ["status", "user_list"]
+    example = {"status": "success","user_list": ["actinia-gdi"]}
+
+
+
+class UserListResponseModel(Schema):
```

Comment:
```suggestion
class UserListResponseModel(Schema):
```

Is this class a dupe of line 1155 etc?

Reply:
ups, I fixed it